src/components/textarea/assets/textarea/t9n/messages.json

```diff
@@ -0,0 +1,4 @@
+{
+  "invalid": "Invalid",
+  "overLimit": "Over the character limit"
```

Comment:
I wonder if there's a better phrase here like "Character limit exceeded"? Do we have someone who can review the copy for this? cc @geospatialem

Reply:
Concur with @driskull's recommendation above ⬆️. 

Also added a note to Figma for discussing support content to support new components with a11y and i18n. cc @ashetland, @SkyeSeitz